tests/ext/pcntl/pcntl_fork_long_running_autoflush.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+Long running autoflush
+--ENV--
+DD_TRACE_GENERATE_ROOT_SPAN=false
+DD_TRACE_AUTO_FLUSH_ENABLED=true
+--FILE--
```

Comment:
All of these tests will need the `--SKIPIF--` sections from #1215 (once it's merged) due to the dependency on `httpbin_integration`. (I haven't merged that PR yet since I noticed some of the tests were inadvertently getting skipped in the PECL tests.)

Reply:
Addressed here https://github.com/DataDog/dd-trace-php/pull/1196/commits/0f8c0ccbe33370f44d57f09607a3224f2c46e84c